packages/react-frontend/components/Review.css

```diff
@@ -0,0 +1,32 @@
+.userReview {
+  display: grid;
+  grid-template-columns: 1fr 2fr;
+  padding: 0.5em;
+  margin: 1em;
+  width: 100%;
+}
+
+.userReview img {
+  height: 120px;
+  width: 120px;
+}
+
+.modifyIcons {
+  float: right;
+}
+
+.modifyIcons > * {
+  padding: 0.3em;
+}
+
+.modifyIcons {
+  color: white;
+  display: flex;
+}
+.reviewContent {
```

Comment:
css looks good! but i think we should also make sure that the words don't overflow, so maybe adding a '.reviewContent h3' section to ensure that the words wrap around?